sapi/fpm/fpm/fpm_log.c

```diff
@@ -205,8 +205,11 @@ int fpm_log_write(char *log_format) /* {{{ */
 							len2 = snprintf(b, FPM_LOG_BUFFER - len, "%.3f", proc.duration.tv_sec + proc.duration.tv_usec / 1000000.);
 						}
 
-					/* miliseconds */
-					} else if (!strcasecmp(format, "miliseconds") || !strcasecmp(format, "mili")) {
+					/* milliseconds */
+					} else if (!strcasecmp(format, "milliseconds") || !strcasecmp(format, "milli") ||
+						   /* mili/miliseconds are supported for backwards compatibility */
+						   !strcasecmp(format, "miliseconds") || !strcasecmp(format, "mili")
```

Comment:
@bukka re https://github.com/php/php-src/pull/6822#discussion_r605716559 this is my attempt to handle both.

I'm not sure what indentation style is desired for this. (How many characters are tabs, and where should I be aiming to get it to line up?)